build.gradle.kts

```diff
@@ -165,6 +165,8 @@ allprojects {
   }
 
   tasks.withType<Test> {
+    useJUnitPlatform()
```

Comment:
Whooops without that, the tests weren't even running 😅 Gradle was apparently looking for Junit 4 tests, found none and considered the execution a success :trollface:

Reply:
lol Xd